arch/risc-v/src/common/espressif/Kconfig

```diff
@@ -1487,6 +1487,24 @@ config ESPRESSIF_FLASH_FREQ_20M
 
 endchoice # ESPRESSIF_FLASH_FREQ
 
+config ESPRESSIF_FLASH_FREQ
+	string
+	# On some of the ESP chips, max boot frequency would be equal to (or even lower than) 80m.
+	# We currently define this to `80m`.
+	default '80m' if ESPRESSIF_FLASH_FREQ_120M
```

Comment:
Why is '120m' not supported?

Reply:
This refers to the flash frequency used to flash the chip. 120m isn't a frequency supported for flashing the device.

Reply:
Just to make it clearer: I didn't change anything regarding flash frequency operation. It's just about the flashing frequency used by `esptool`. 120 MHz isn't a valid frequency for flashing the device.

Can you please take a second look at the PR, @acassis?